File: app/src/main/java/com/pkmnapps/activitydo/custominterfaces/HomeFragInterace.java
package com.pkmnapps.activitydo.custominterfaces;

import com.pkmnapps.activitydo.dataclasses.ActivityData;

public interface HomeFragInterace {

    void displayEditDialog(final ActivityData activityData);
    void displayDeleteDialog(final ActivityData activityData);
    void changeColorTheme(int pos);
    void updatePinnedMenu();
}

File: app/src/main/java/com/pkmnapps/activitydo/custominterfaces/HomeFragInterace.kt
package com.pkmnapps.activitydo.custominterfaces

import com.pkmnapps.activitydo.dataclasses.ActivityData
import org.junit.runner.RunWith

interface HomeFragInterace {
    open fun displayEditDialog(activityData: ActivityData?)
    open fun displayDeleteDialog(activityData: ActivityData?)
    open fun changeColorTheme(pos: Int)
    open fun updatePinnedMenu()
}